src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -229,6 +235,41 @@ module approveKeyVaultPrivateEndpointConnections 'keyVaultEndpoints.bicep' = {
   }
 }
 
+resource dataExplorerManagedPrivateEndpoint 'Microsoft.DataFactory/factories/managedVirtualNetworks/managedPrivateEndpoints@2018-06-01' = if (deployDataExplorer) {
+  name: dataExplorerName
+  parent: managedVirtualNetwork
+  properties: {
+    name: dataExplorerName
+    groupId: 'cluster'
+    privateLinkResourceId: dataExplorerId
+    fqdns: [
+      dataExplorerUri
+    ]
+  }
+}
+
+
+module getdataExplorerPrivateEndpointConnections 'dataExplorerEndpoints.bicep' = if (deployDataExplorer) {
+  name: 'GetdataExplorerPrivateEndpointConnections'
+  dependsOn: [
+    stopTriggers
+  ]
+  params: {
+    dataExplorerName: dataExplorerName
+  }
+}
+
+module approvedataExplorerPrivateEndpointConnections 'dataExplorerEndpoints.bicep' = if (deployDataExplorer) {
+  name: 'ApprovedataExplorerPrivateEndpointConnections'
+  dependsOn: [
+    getdataExplorerPrivateEndpointConnections
+  ]
+  params: {
+    dataExplorerName: dataExplorerName
+    privateEndpointConnections: getdataExplorerPrivateEndpointConnections.outputs.privateEndpointConnections
+  }
+}
```

Capitalizing the "d" in "DataExplorer"
```suggestion
module approveDataExplorerPrivateEndpointConnections 'dataExplorerEndpoints.bicep' = if (deployDataExplorer) {
  name: 'ApproveDataExplorerPrivateEndpointConnections'
  dependsOn: [
    getDataExplorerPrivateEndpointConnections
  ]
  params: {
    dataExplorerName: dataExplorerName
    privateEndpointConnections: getDataExplorerPrivateEndpointConnections.outputs.privateEndpointConnections
  }
}
```